src/plugins/kibana/server/lib/ingest_process_response.js

```diff
@@ -0,0 +1,50 @@
+const _ = require('lodash');
```

Comment:
I'd love to see some tests for this file with expected inputs and outputs. Documentation in the PR description for the response payload schema would also be extremely helpful.